file-share-service-desktop-client/Modules/Auth/AuthenticateView.xaml

```diff
@@ -8,6 +8,15 @@
              mc:Ignorable="d"
              d:DesignHeight="450" d:DesignWidth="800"
              d:DataContext="{d:DesignInstance Type=auth:AuthenticateViewModel, IsDesignTimeCreatable=False}">
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <Style TargetType="ComboBox" x:Key="DropBox">
```

Comment:
This style isn't actually used. Could we remove it please. (and then, the ResourceDictionary element).

Reply:
Done the required changes Kindly recheck